src/framework/dropdown/DropdownGroup.jsx

```diff
@@ -0,0 +1,41 @@
+
+import classNames from 'classnames';
+import React, {
+  PropTypes,
+} from 'react';
+
+import Dropdown from './Dropdown.jsx';
+
+const DropdownGroup = (props) => {
+  const options = props.options.reduce(
+    (accumulator, optionList, currentIndex) => (
+      accumulator.concat(optionList.map((option, index) => (
+        Object.assign({}, option, {
+          classes: classNames(props.optionGroupClasses[currentIndex], {
+            'dropdownGroup--first': (accumulator.length > 0 && index === 0),
+          }),
+        })
+      )))
+    ),
+  []);
+
+  const extendedProps = Object.assign({}, props, {
+    options,
+  });
+
+  return (
+    <Dropdown
+      {...extendedProps}
+    />
+  );
+};
+
+DropdownGroup.propTypes = Object.assign({}, Dropdown.propTypes, {
+  optionGroupClasses: PropTypes.array,
```

Comment:
Optionally, there's no harm if we allow people to define the class for separator as well when we already allow each group to have a different class. Up to you.

Reply:
I though about that, but it can also be controlled by the group class, in the example you can do 
```
.dropdownLabel--red.dropdownGroup--first {
    border-color: red;
}
```
to change the border color of the separator